test/Imports.test.kt

```diff
@@ -31,4 +31,17 @@
 //                                         ^ source.kotlin meta.import.kotlin
 //                                          ^^ source.kotlin meta.import.kotlin keyword.other.kotlin
 //                                            ^ source.kotlin meta.import.kotlin
-//                                             ^^^^ source.kotlin meta.import.kotlin entity.name.type
\ No newline at end of file
+//                                             ^^^^ source.kotlin meta.import.kotlin entity.name.type
+
+  import com.google.test.CommentedClass // comment
+//^^^^^^ source.kotlin meta.import.kotlin keyword.other.import.kotlin
+//      ^ source.kotlin meta.import.kotlin
+//       ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ source.kotlin meta.import.kotlin storage.modifier.import.kotlin
+//                                      ^^ source.kotlin punctuation.definition.comment.kotlin comment.line.double-slash.kotlin
+
+  import com.google.test.CommentedClass /*comment*/ as Another
```

Comment:
Looks like it detects the start and end of the comment here but doesn't include scopes for the `as Another` portion of the import.